tests/ha/run.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Do you forget to add this case in _tests/others_integration.txt_?

Reply:
yes, I add it in 34cd486